Server/src/diagramcoordinator.erl

```diff
@@ -2,17 +2,20 @@
 -export([init/3]).
 
 %%Author: Tim Jonasson
-%%Version: 1.3
+%%Collaborator: Isabelle Törnqvist 2017-10-30
+%%Collaborator: Kosara Golemshinska 2017-11-31
+%%Version: 1.6
 
 %Returns no_classes when there was no classes in the given diagram 
 init(_Usercoordinator, _Did, {[], _}) -> no_classes;
 %Returns no_messages when there was no messages in the given diagram 
 init(_Usercoordinator, _Did, {_, []}) -> no_messages;
 %Spawns and Initializes the diagram coordinator
 init(Usercoordinator, Did, {L, Messages}) -> 
-  Pids = spawn_nodes(L),
-  loop(Usercoordinator, Did, Pids, Messages, 1).
-  
+	%Sending information that the usercoordinator has been spawned. To be printed in the executionlog
```

Comment:
These are changes done for the executionlog. what will happen when merging this?

Reply:
I didn't change this, this was taken from develop

Reply:
ah oki